### DISCUSSION TITLE
Issue booting from NVME in Orin Nano with JNX42

### DISCUSSION THREAD

**Discussion by jsalas98** [OE4T/meta-tegra#1436] [Q&A] [answered]
Hi everyone,

I’m using the Auvidea’s JNX42 carrier board for an Orin Nano and I’ve been having issues after flashing since the device won’t even show the NVIDIA logo when I run the` initrd_flash` script. The flashing logs don’t show any noticeable error and I don’t have access to the serial console log because the carrier board doesn’t expose it so I’m kinda blind in that sense.

So far to support this board I just updated the kernel device tree option in my Yocto config. I’m using the jetson-orin-nano-devkit-nvme as base config.

What I have found out until now is that it may be related to something on the NVME flashing because when I flash the qspi using `doflash.sh —spi-only` and then flash the NVME with the SDK manager l4t initrd script using the external only options, it boots correctly. But when I flash the NVME using `initrd_flash.sh —skip-bootloader `(or without skipping bootloader) it won’t even show the NVIDIA logo anymore. I would have thought that the boot options screen was completely decoupled from the partitions on the external device but because of this test it seems flashing the NVME is messing with something on the early boot stages.

Any ideas of what could be going on would be really appreciated.

Thanks!

**Comment by madisongh** [comment, marked as answer]
I took a quick look at what Auvidea provides in their software kit for the JNX42, and besides the kernel device tree changes, they also modify the `tegra234-mb2-bct-misc-p3767-0000.dts` file used by the early stage bootloader, to set `cvb_eeprom_read_size` to 0.  Based on other discussions here (#1171 and #1284), this looks like a fairly common sticking point with third-party carriers.
